src/AppBundle/Tests/Entity/AliasTest.php

```diff
@@ -0,0 +1,37 @@
+<?php
+
+namespace AppBundle\Tests\Entity;
+
+use AppBundle\Entity\Alias;
+use AppBundle\Entity\Person;
+use PHPUnit_Framework_TestCase;
+
+class AliasTest extends PHPUnit_Framework_TestCase {
+    
+    public function testAppendNote(){
+        
+        $alias = new Alias();
+        $testNote = "This is a note to append.";
+        
+        $alias->appendNote($testNote);
+        $this->assertEquals($testNote, $alias->getNotes());
+    }
+    
```

Comment:
There is already a LoadAlias data fixture. To use it, you should change the class to extend BaseTestCase and add a getFixtures method to the AliasTest that looks like this:

```php
    protected function getFixtures() {
        return array(
            LoadAlias::class,
        );
    }
```

Then, in the test function instead of `$alias = new Alias();` you would do ` $alias = $this->references->getReference('alias.1');`

It might seem like a trivial change, and for something this simple is probably is. But if you need to start testing books and authors and the relationships between them it becomes much much easier to have a different piece of code create the entities for you.

Reply:
Not sure I understand what's going on, but solved!